db/migrate/20190212111202_create_policies.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePolicies < ActiveRecord::Migration
+  def change
+    create_table :policies do |t|
```

Comment:
@hallelujah @davidor Should it belong to a `proxy` or to a `service` or to a `provider` or to something? 😂 Plus, should it have the `tenant_id`?

Reply:
David said that the list of policies must be unique per tenant 👀 So at least it must belong to a tenant for sure, but I want to know if this has more implications with other models 😄

Reply:
Yes we need `tenant_id` (with proper trigger) and `account_id` columns

Reply:
It does not belong to a proxy nor a service just a provider (Account)